aptos-move/move-examples/cli-e2e-tests/common/sources/cli_e2e_tests.move

```diff
@@ -0,0 +1,327 @@
+module addr::cli_e2e_tests {
```

Comment:
I am a bit confused as for why we need this move module? python has a compiled module we use for testing https://github.com/aptos-labs/aptos-core/blob/main/ecosystem/python/sdk/examples/two_by_two_transfer.mv can't we use this one to test the `aptos move` subcommands?

Reply:
Well the point of these tests is to be end to end, so we do the same steps the users would do. No one really precompiles modules, takes the mv file, and then publishes that.

Reply:
We should likely then, put one place where this can be used across multiple test suites, including somehow autocompiling it for the Python SDK.